tests/components/samsungtv/test_media_player.py

```diff
@@ -163,42 +165,50 @@ def mock_now():
     return dt_util.utcnow()
 
 
-async def setup_samsungtv(hass, config):
+async def setup_samsungtv(hass: HomeAssistantType, config: dict):
     """Set up mock Samsung TV."""
     await async_setup_component(hass, SAMSUNGTV_DOMAIN, config)
     await hass.async_block_till_done()
 
 
-async def test_setup_with_turnon(hass, remote):
+async def test_setup_with_turnon(hass: HomeAssistantType, remote: Mock):
     """Test setup of platform."""
     await setup_samsungtv(hass, MOCK_CONFIG)
     assert hass.states.get(ENTITY_ID)
 
 
-async def test_setup_without_turnon(hass, remote):
+async def test_setup_without_turnon(hass: HomeAssistantType, remote: Mock):
     """Test setup of platform."""
     await setup_samsungtv(hass, MOCK_CONFIG_NOTURNON)
     assert hass.states.get(ENTITY_ID_NOTURNON)
 
 
-async def test_setup_websocket(hass, remotews, mock_now):
+async def test_setup_websocket(hass: HomeAssistantType, remotews: Mock, mock_now: Mock):
     """Test setup of platform."""
-    with patch("homeassistant.components.samsungtv.bridge.SamsungTVWS") as remote_class:
+    with patch(
+        "homeassistant.components.samsungtv.bridge.SamsungTVWS"
+    ) as remote_class, patch(
+        "homeassistant.components.samsungtv.config_flow.gethostbyname"
+    ):
         enter = Mock()
         type(enter).token = PropertyMock(return_value="987654321")
         remote = Mock()
         remote.__enter__ = Mock(return_value=enter)
         remote.__exit__ = Mock()
+        remote.rest_device_info.return_value = {"device": {"type": "Samsung SmartTV"}}
         remote_class.return_value = remote
 
         await setup_samsungtv(hass, MOCK_CONFIGWS)
 
-        assert remote_class.call_count == 1
-        assert remote_class.call_args_list == [call(**MOCK_CALLS_WS)]
+        assert remote_class.call_count == 2
+        assert remote_class.call_args_list == [
+            call(**MOCK_CALLS_WS),
+            call(**MOCK_CALLS_DEVICEINFO),
+        ]
         assert hass.states.get(ENTITY_ID)
 
 
-async def test_setup_websocket_2(hass, mock_now):
+async def test_setup_websocket_2(hass: HomeAssistantType, mock_now: Mock):
```

Comment:
```suggestion
async def test_setup_websocket_2(hass: HomeAssistant, mock_now: Mock):
```

No need to use the Type since its just an alias for `HomeAssistant`

Reply:
Done